src/main.rs

```diff
@@ -41,20 +40,11 @@ fn main() -> Result<()> {
     // build Docker Image
     // FIXME this assumes that this process is run from the docker/ directory
     fs::write("Dockerfile", String::from(
-    r#"# Although I still don't really get why we have to recompile it
-# Why we shouldn't use the edge image?
-#   *  Because that might be a point of failure for eg what happends when the image is not pushed
-#      that would lead to a previous commit being used. But we can add an additional check that
-#      makes sure whatever commit is used to build the image is same as the commit received from
-#      webhooks so that isn't really a problem.
-# Why we should use the edge image?
-#   *  Because that would prevent us from recompiling the project again and again. We currently
-#      recompile tremor a ton of times in tremor and it would be great if we could avoid that.
-FROM rust:1.52.1 as builder
+    r#"FROM rust:1.53 as builder
```

Comment:
we should use `inclide_str!` here to load the file https://doc.rust-lang.org/std/macro.include_str.html